cmd/kubeadm/app/preflight/checks.go

```diff
@@ -81,16 +81,17 @@ type ServiceCheck struct {
 }
 
 func (sc ServiceCheck) Check() (warnings, errors []error) {
+	warnings, errors = []error{}, []error{}
```

Comment:
Yes, better to return nil if no data. Maybe it is enough to just remove this line.